src/dataviews/histogram-dataview/histogram-data-model.js

```diff
@@ -20,17 +20,30 @@ module.exports = Model.extend({
 
   url: function () {
     var params = [];
+    var columnType = this.get('column_type');
     var offset = this._getCurrentOffset();
+    var aggregation = this.get('aggregation');
 
-    if (this.get('column_type') === 'number' && this.get('bins')) {
+    if (columnType === 'number' && this.get('bins')) {
       params.push('bins=' + this.get('bins'));
-    } else if (this.get('column_type') === 'date') {
-      params.push('aggregation=' + (this.get('aggregation') || 'auto'));
+    } else if (columnType === 'date') {
+      params.push('aggregation=' + (aggregation || 'auto'));
```

Comment:
What about `var aggregation = this.get('aggregation') || 'auto;` and simplify this line?